shared-bindings/displayio/FourWire.c

```diff
@@ -103,33 +103,59 @@ STATIC mp_obj_t displayio_fourwire_make_new(const mp_obj_type_t *type, size_t n_
     return self;
 }
 
-//|   .. method:: send(command, data)
+//|   .. method:: reset()
+//|
+//|     Performs a hardware reset via the reset pin. Raises an exception if called when no reset pin
+//|     is available.
+//|
+STATIC mp_obj_t displayio_fourwire_obj_reset(mp_obj_t self_in) {
+    displayio_fourwire_obj_t *self = self_in;
+
+    if (!common_hal_displayio_fourwire_reset(self)) {
+        mp_raise_RuntimeError(translate("no reset pin available"));
+    }
+    return mp_const_none;
+}
+MP_DEFINE_CONST_FUN_OBJ_1(displayio_fourwire_reset_obj, displayio_fourwire_obj_reset);
```

Comment:
What is the reasoning here behind rising an exception rather than just doing nothing? It forces us to repeat that check for pin everywhere (or catch the exception). Is there are use case where that exception is useful?